core/shape-parts.cpp

```diff
@@ -39,6 +39,32 @@ const Path& BasicBezier::generate(Point endPoint, Point transitionPoint) {
   return m_path;
 }
 
+const Path& FlatBezier::generate(Point endPoint, Point transitionPoint) {
+  MCFixed flatEndY = m_flatWidth / -2; // WHY DO I HAVE TO MAKE THIS NEGATIVE TO GET IT TO WORK???
+  if (endPoint.X < transitionPoint.X) {
+    Path ep;
+    ep.push_back(endPoint);
+    m_path = BezierPath(Point(endPoint.X, flatEndY),
+                        Point(endPoint.X, flatEndY + 
```

Comment:
This is the Bezier control point to pull upwards on the curve from the center of the nose. It should be (0, 12 * `m_endHandle`) or (0, 10 * `m_endHandle`). You can kinda pick whether you what this control point to cover the distance from the center of the nose or the end of the flat section. It doesn't matter much, I might suggest you have it cover the entire space for simplicity, i.e., (0, `(transitionPoint.Y - endPoint.Y) * m_endHandle)`).